test/integ_tests/conftest.py

```diff
@@ -162,15 +162,21 @@ def job_failed_name(request):
 
 @pytest.fixture(scope="session", autouse=True)
 def completed_quantum_job(aws_session, job_completed_name):
```

Comment:
`aws_session` no longer needed
```suggestion
def completed_quantum_job(job_completed_name):
```